docs/releases.md

```diff
@@ -1,23 +1,45 @@
 ## Releases
 
-## Preparation
-
-To **prepare the extension for a new version**, you must:
-
-- update and commit the version in `package.json`
-- update and commit the `CHANGELOG.md`
-- tag, push, and merge to the `master` branch on GitHub
-
-## Packaging
-
-To **build a signed extension .xpi**, you must: commit and push to the
-`production` branch (ideally as a merge commit from `master`) on GitHub.
-
-## Extension Signing
+### Instructions
+
+1. Update "version" in package.json (and package-lock.json)
+  - we follow the [semver](http://semver.org/) syntax
+  - Alpha releases will be tagged with "-alpha" (e.g., "0.1.0-alpha")
+  - Beta releases will be tagged with "-beta" (e.g., "0.x.0-beta")
+  - Stable releases will **not** be tagged, and follow smever from the last Beta release (major, minor, and patch)
+2. Update "CHANGELOG.md" using `conventional-changelog`
```

Comment:
After talking with @sandysage , I think the right thing to do is:

* write the CHANGELOG.md (release-notes.md) by hand, based on the issue board (waffle.io and/or trello)
* make sure it has sections for "what's new", "bugs fixed", "known issues" (see PR #176 )

The rationale here is that changelog/release notes should be written for the intended audience of the product.  This is effectively an application, so we should have release notes that users can understand.